examples/ts-node/src/index.ts

```diff
@@ -1,42 +1,35 @@
 import { SubspaceClient, Identity } from "subspace.js";
-import { mnemonicGenerate } from "@polkadot/util-crypto";
-import * as fs from "fs";
-import * as dotenv from "dotenv";
-dotenv.config();
+import { promises as fs } from "fs";
 
-const NODE_WS_PROVIDER = process.env.NODE_WS_PROVIDER;
-const FARMER_WS_PROVIDER = process.env.FARMER_WS_PROVIDER;
-const mnemonic = mnemonicGenerate();
+const NODE_WS_PROVIDER = "ws://localhost:9944";
+const FARMER_WS_PROVIDER = "ws://localhost:9955";
+const suri = "//Alice";
 
-fs.readFile("./sample-input.jpg", {}, async (err, data) => {
-  if (err) throw err;
+(async () => {
+  const data = await fs.readFile("./sample-input.jpg");
   await usage(new Uint8Array(data));
-});
+  process.exit(1);
+})();
```

Comment:
Now let's embed `usage` function right into here and remove `process.exit(1)` since there is no error.

BTW, with latest Node.js top-level await should work, so you don't have to use IIFE if TS already supports that, will look nicer.

Reply:
I made some changes to get top-level await working. 

[#b4bdb42fcb484bea6e85055756d8264638231500](https://github.com/subspace/subspace.js/pull/4/commits/b4bdb42fcb484bea6e85055756d8264638231500)